docker/README.md

```diff
@@ -0,0 +1,29 @@
+## Docker usage
+
+We provide two dockerfiles for CPU and GPU respectively. For CPU users, we install MMDeploy with ONNXRuntime, ncnn and OpenVINO backends. For GPU users, we install MMDeploy with TensorRT backend. Besides, users can install mmdeploy with different versions when building the docker image.
+
+### Build docker image
+
+For CPU users, we can build the docker image with the latest MMDeploy through:
+```
+cd mmdeploy
+docker build docker/CPU/ -t mmdeploy:master
```

Comment:
Maybe image tags could be?
For GPU: mmdeploy:master -> mmdeploy:gpu-master
For CPU: mmdeploy:master -> mmdeploy:cpu-master

Reply:
Thanks for the suggestion. Will use `mmdeploy:master-gpu` :smiley: